cli/command/service/opts_test.go

```diff
@@ -326,3 +328,142 @@ func TestToServiceSysCtls(t *testing.T) {
 	assert.NilError(t, err)
 	assert.Check(t, is.DeepEqual(service.TaskTemplate.ContainerSpec.Sysctls, expected))
 }
+
+func TestToPrivilegesAppArmor(t *testing.T) {
+	for _, mode := range []string{"default", "disabled"} {
```

Comment:
Wrap in subtests so it'll be easier to tell from the test logs which case failed.